docs/command-reference/server-management/info.md

```diff
@@ -146,7 +144,7 @@ Here is the meaning of all fields in the **clients** section:
 
 *   `connected_clients`: Number of client connections (excluding connections
      from replicas)
-*    `client_read_buf_capacity`: TODO
+*   `client_read_buf_capacity`: Client read buffer size in bytes
```

Comment:
here we probably did a bad job naming this metric.
it's in fact the overall capacity of read buffers of all clients. the metric can go up and down.
maybe to rename it to `client_read_buffer_bytes` ?

Reply:
Updated to `dragonfly_version` and `client_read_buffer_bytes`.